vue-frontend/src/components/Home.vue

```diff
@@ -3,17 +3,20 @@
     <!-- Main Website -->
     <div class="main-wrapper">
       <Header />
```

Comment:
Shouldn't we make header section sticky after scroll?

Reply:
Not now, let's add tickets for everything related to animation. We want to implement quick return pattern for website.